frontend/src/store/index.js

```diff
@@ -1498,9 +1498,25 @@ const mutations = {
     state.splitpaneResize = value
   },
   SET_DARK_MODE (state, value) {
-    state.darkMode = value
-    localStorage.setItem('global/dark-mode', value)
-    Vue.vuetify.framework.theme.dark = value
+    if (value !== undefined) {
```

Comment:
From my point of view this is an antipattern. A mutation should only mutate the store state and not run mediaqueries.  The new value must come from outside the mutation. Why do we need two properties `darkMode` and `darkTheme`.

Reply:
We should start using store plugins for this kind of problems.
```js
export default function createMediaPlugin (window) {
  return store => {
    const mq = window.matchMedia('(prefers-color-scheme: dark)')
    store.commit('SET_PREFERS_COLOR_SCHEME', mq.matches ? 'dark' : 'light')
    mq.addListener(e => {
      store.commit('SET_PREFERS_COLOR_SCHEME', e.matches ? 'dark' : 'light')
    })
  }
}
```